deployment/terraform/agent.go

```diff
@@ -243,3 +232,27 @@ func (t *Terraform) initLoadtest(extAgent *ssh.ExtAgent, initData bool) error {
 
 	return nil
 }
+
+func (t *Terraform) getAppHostsFile(index int) (string, error) {
+	output, err := t.Output()
+	if err != nil {
+		return "", err
+	}
+
+	// The new entry in /etc/hosts will make SiteURL point to:
+	// - The first instance's IP if there's a single node
+	// - The proxy's IP(s) if there's more than one node
```

Comment:
Can we specify here what "node" means? And maybe indicate that we  It's starting to get confusing :joy: 
```suggestion
	// - The first instance's IP if there's a single app node
	// - The IP of one of the proxy nodes if there's more than one app node
```